ingeniamotion/wizard_tests/base_test.py

```diff
@@ -52,6 +53,17 @@ def restore_backup_registers(self):
                 except IMRegisterWrongAccess as e:
                     self.logger.warning(e, axis=subnode)
 
+    @Stoppable.stoppable
+    def show_error_message(self):
+        error_code, axis, warning = self.mc.errors.get_last_buffer_error(
+            servo=self.servo, axis=self.axis)
+        # TODO check if clan warning bit is necessary
+        error_code_cleaned = error_code & self.WARNING_BIT_MASK
```

Comment:
It should not be necessary, in get_last_buffer_error function, the error code hasn't warning bit. Check it, please.